.github/workflows/clp-core-build.yaml

```
@@ -95,9 +124,79 @@ jobs:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: "x86_64"
+            os_name: "centos-stream-9"
+          - arch: "aarch64"
+            os_name: "manylinux_2_28-aarch64"
+          - arch: "x86_64"
+            os_name: "manylinux_2_28-x86_64"
+    name: "${{ matrix.os_name }}-deps-image"
+    if: >-
```

actionlint (1.7.7)

context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)